app/views/admin/invitations/index.html.erb

```diff
@@ -56,7 +56,7 @@
     </div>
 
   </header>
-  <table class="hidden m:table crayons-table crayons-table--minimal" width="100%">
+  <table class="hidden m:table crayons-table crayons-table--minimal" width="100%" aria-describedby="empty-state-message-id">
```

Comment:
```suggestion
  <table class="hidden m:table crayons-table crayons-table--minimal" width="100%" <%= @invitations.empty? ?  ' aria-describedby="empty-state-message-id"'.html_safe : '' %>">
```

I don't think there's any negative impact, but we'll see errors in things like axe dev tools if we reference IDs that don't exist on the page (which will be the case when the empty state message isn't shown)

Reply:
This is a great suggestion; thanks, @aitchiss!